packages/victory-area/src/area.js

```diff
@@ -93,7 +93,25 @@ const getAreaFunction = (props) => {
     : getCartesianArea(props, interpolation);
 };
 
+const evaluateProps = (props) => {
+  /**
+   * Potential evaluated props are
+   * 1) `desc`
+   * 2) `id`
+   * 3) `style`
+   * 4) `tabIndex`
+   * 5) everything else
```

Comment:
Everything else?

Reply:
I just copied this comment from a different component, I can be more explicit about it.

Reply:
I think you can skip everything else if you're listing all the props that are evaluated.

Reply:
ok cool. thanks